classes/maps.php

```diff
@@ -218,7 +218,7 @@ function generate_iframe() {
 						}</style>
 					<script
 						src="<?php echo esc_url( SIMPLEMAP_MAPS_JS_API . '?v=3&amp;sensor=false&amp;language=' . $atts['default_language'] . '&amp;region=' . $atts['default_country'] ); ?>"
-						type="text/javascript"></script>
+						type="text/javascript"></script>	
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->